hack/bundle-gen.py

```diff
@@ -358,6 +366,8 @@ def open_pr(
     dest_github_org = upstream_repo.split(":")[1].split("/")[0]
     dest_github_reponame = dir_name
 
+    update_channel = CHANNEL_DEFAULT
```

Comment:
I'd be okay if we got rid of the `update_channel` var entirely and just hardcoded it in the two remaining spots below.